scipy/io/matlab/_mio.py

```diff
@@ -274,6 +274,7 @@ def savemat(file_name, mdict,
     Examples
     --------
     >>> from scipy.io import savemat
+    >>> import numpy as np
```

Comment:
Should this line be at the top of the code block based on this [comment](https://github.com/scipy/scipy/issues/13049#issuecomment-1199457608)?

Reply:
I'll make the necessary changes then.

Reply:
I've made changes to `scipy.optimize` . However, changes to `scipy.io` have already been merged. So should I change it or leave it as is?

Reply:
Do you mean this change already includes in the master branch? If so, please rebase this PR's branch on master.

Reply:
Thank you for the feedback, I've removed changes in the io module from this PR.

Reply:
I've made the changes according to your review.